index.html

```diff
@@ -189,7 +190,7 @@ <h2>About Me</h2>
 				</div>
 			</div>
 
-			<div class="col-md-1"></div>
+			<div className="col-md-1"></div>

```

Comment:
In HTML, the attribute for defining classes is "class" not "className". Update it to `<`div class="col-md-1"`>``<`/div`>`.